pyproject.toml

```diff
@@ -95,6 +96,8 @@ module = [
   'sphinx_needs.functions.common',
   'sphinx_needs.functions.functions',
   'sphinx_needs.layout',
+  'sphinx_needs.lsp.esbonio',
+  'sphinx_needs.lsp.needs_store',
```

Comment:
new modules should not be added to the mypy whitelist. This is for pre-existing modules that are not (yet) "type-safe".